### ISSUE TITLE
Purpose of your request domain screen reader output

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2914]
![Screenshot 2024-10-07 at 4 20 10 PM](https://github.com/user-attachments/assets/8bd9ffba-c595-439d-866a-e33a6180ba7d)

### Current screen reader output
Purpose * You can enter up to 2000 characters required

`<label>`: Purpose *
`aria-describedby`: You can enter up to 2000 characters
`required`: required

### What it should be
**Default:**
What is the purpose of your requested domain? Describe how you’ll use your .gov domain. Will it be used for a website, email, or something else? You can enter up to 2000 characters required

`<legend>:` What is the purpose of your requested domain?
`<label>`: Describe how you’ll use your .gov domain. Will it be used for a website, email, or something else? 
`aria-describedby`: You can enter up to 2000 characters
`required`: required

**Error state:** 
What is the purpose of your requested domain? Describe how you’ll use your .gov domain. Will it be used for a website, email, or something else? You can enter up to 2000 characters required, Describe how you’ll use the .gov domain you’re requesting. 
`required`: required, Describe how you’ll use the .gov domain you’re requesting. 

### Prioritization Comments
**WCAG Conformance Level:** 2.1 AA
**WCAG Standard that applies:** 
1.3.1 - Info and Relationships (Critical, partially supported)
3.3.2 - Labels or Instructions (Critical, partially supported)
4.1.2 - Name, Role, Value (Critical, partially supported)
**Critical Failure if Not Met?:**  No
**Prioritization:** High/Must Do
**Reason:** This issue impacts accessibility by providing slightly incorrect labeling and descriptions for screen reader users. However given the context of this issue, the screenreader does say "required" in the ARIA output so this is not a complete failure. Ensuring accessible and descriptive labels is essential for clear form navigation, especially for users relying on assistive technologies.

**Comment by erinysong** [comment]
How important is it that we reach this screenreader output from a legend? A legend typically indicates that we group together related field and are traditionally grouped with a box around them. We can style our CSS to remove this but given this form only has one field, would we rather just add screenreader text indicating the details above without changing the HTML to a legend?
